lib/inspec/reporters/json.rb

```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+
+require 'json'
+
+module Inspec::Reporters
+  class Json < Base
+    def render
+      output clean_json.to_json
+    end
+
+    private
+
+    def clean_json
```

Comment:
The way this is implemented leaves us open to regressions if someone adds additional fields to the base formatter.

Since there's an associated schema, I'd rather see us actually build the object to match the schema rather than delete fields. That way if additional fields are added, they don't end up in this output.